server/users/user-setting-model.ts

```diff
@@ -0,0 +1,21 @@
+import { DomainModel } from "../core/domain.model"
+export class UserSettingModel implements DomainModel {
+	
+	public readonly description: string
+	public readonly value: boolean
+	public readonly column_name: string
```

Comment:
`camelCase`

We're only using snake case in generated DB models, but only accidentally, I'll change that for consistency.

Reply:
Resolved in https://github.com/mbrizic/ratethegame/pull/28/commits/0713f38e2625f1f98f02c12982914cdc2f903b6e